js/views/add_server_dialog_view_old.js

```diff
@@ -0,0 +1,92 @@
+/* global Whisper, i18n, _ */
+
+// eslint-disable-next-line func-names
+(function() {
+  'use strict';
+
+  window.Whisper = window.Whisper || {};
+
+  Whisper.AddServerDialogView = Whisper.View.extend({
+    templateName: 'add-server-template',
+    className: 'loki-dialog add-server modal',
+    initialize(options = {}) {
+      this.title = i18n('addServerDialogTitle');
+      this.okText = options.okText || i18n('ok');
+      this.cancelText = options.cancelText || i18n('cancel');
+      this.$('input').focus();
+      this.render();
```

Comment:
entire file must be removed isn't it ? Its name is _old.js

Reply:
Done